web-app/django/VIM/settings.py

```diff
@@ -134,8 +135,17 @@
 STATIC_URL = "static/"
 STATICFILES_DIRS = [
```

Comment:
I get the following warnings on Django startup:

2025-05-22 15:05:23 ?: (staticfiles.W004) The directory '/virtual-instrument-museum/frontend/assets' in the STATICFILES_DIRS setting does not exist.
2025-05-22 15:05:23 ?: (staticfiles.W004) The directory '/virtual-instrument-museum/vim-app/VIM/static' in the STATICFILES_DIRS setting does not exist.

Reply:
I suspect this has to do with the changes to the directory structure under `web-app`